official/utils/misc/distribution_utils.py

```diff
@@ -56,6 +60,10 @@ def get_distribution_strategy(distribution_strategy="default",
                        "cannot be set to 'off'.".format(num_gpus))
     return None
 
+  if distribution_strategy == "collective" or num_workers > 1:
```

Comment:
@yuefengz @dubey Since the name of the strategy as exposed has now changed (MultiWorkerMirroredStrategy), i wonder if we should rename this flag too?

Reply:
done.  I'll change the strategy object from contrib.distribute.collectiveallreducestrategy to distribute.multiworkermirroredstrategy in a follow-on PR.